fluent-plugin-google-cloud.gemspec

```diff
@@ -21,9 +21,9 @@ eos
 
   gem.add_runtime_dependency 'fluentd', '~> 0.10'
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
-  gem.add_runtime_dependency 'google-api-client', '> 0.9'
+  gem.add_runtime_dependency 'google-api-client', '> 0.9', '< 0.10'
```

Comment:
I believe `~>` goes to the highest version in that level. So if we did `~>0.9' it would go to version `0.9` and above e.g. `0.10`,`0.11`.. but not to `1.0`.